src/components/LeftSidebar/LeftSidebar.vue

```diff
@@ -277,6 +287,25 @@
 
 		<template #footer>
 			<div class="left-sidebar__settings-button-container">
+				<NcButton v-if="isFiltered"
+					type="tertiary"
+					wide
+					@click="handleFilter(null)">
+					<template #icon>
+						<FilterRemoveIcon :size="20" />
+					</template>
+					{{ t('spreed', 'Clear filters') }}
+				</NcButton>
+				<NcButton v-else
+					type="tertiary"
+					wide
+					@click="handleFilter('archived')">
+					<template #icon>
+						<IconArchive :size="20" />
+					</template>
+					{{ t('spreed', 'Archived conversations') }}
+				</NcButton>
```

Comment:
I am not sure if we need another clear filter button here.
Yet, Archived conversations should be shown ONLY if there are archived ones indeed.

I think you put filter button here so it undoes what archived conversation button do? in that case, we show `return to conversations` or sthg like that (we don't show to user this feature as a filter but as another page/tab in left sidebar ;) ). Also, we won't show it in filters action.